src/main/resources/i18n/MessageBundle.properties

```diff
@@ -8,6 +8,8 @@ command.input.file.arguments = global.json> <month.json> <output-file.tex
 command.output.version = Version: {0}
 command.output.buildInfo = Built from {0} ({1}) at {2}
 
+gui.errorListTitle = Error
```

Comment:
Maybe we should change this to gui.errorPaneTitle in order to avoid misunderstandings. I think the name "pane" or "window" is clearer for a GUI component's title than "list".

Reply:
"list" was not referencing the GUI, but the fact that a list of errors is displayed. How about "errorListWindowTitle"? :D

Reply:
I see. At the moment it's just used for the GUI title therefore it would be great to also make that directly visible in the MessageBundle. "errorListWindowTitle" completely does that :)